tas_car:
  # Publish all joint states -----------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Wheel Controllers      -------------------------------
  r_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_r_b_wheel_joint
    pid: {p: 0.3, i: 0.005, d: 0.005}  # Tune with caution! High values might lead to model collapse!
  
  l_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_l_b_wheel_joint
    pid: {p: 0.3, i: 0.005, d: 0.005}  # Tune with caution! High values might lead to model collapse!

  # Ackermann steering controllers -----------------------
  r_steer_controller:
    type: effort_controllers/JointPositionController
    joint: r_ack_joint
    pid: {p: 5.0, i: 0.05, d: 0.5}

  l_steer_controller:
    type: effort_controllers/JointPositionController
    joint: l_ack_joint
    pid: {p: 5.0, i: 0.05, d: 0.5}
